run_process_zephir_incremental.sh

```diff
@@ -22,6 +22,23 @@ cd $TMPDIR
 SCRIPTNAME=`basename $0`
 zephir_date="$(echo $YESTERDAY | sed 's/\(....\)\(..\)/\1-\2-/')"
 
+# Route all external processes through these functions
+# to avoid silent failures.
+function run_external_command {
+  $1
+  cmdstatus=$?
+  if [ $cmdstatus != "0" ]; then
+    message="`date`: error: '$1' returned $cmdstatus"
+    report_error_and_exit "$message"
+  fi
+}
+
+function report_error_and_exit {
+  echo $1
```

Comment:
I would suggest `echo "$1"` to preserve the quoting in the incoming parameter (but we don't need `"$@"` here since we have a single message rather than something intended to be multiple arguments